holidays/countries/honduras.py

```diff
@@ -71,7 +71,7 @@ def _populate(self, year):
             # (First Wednesday of October from 12 noon to Saturday 12 noon)
             holiday_name = "Semana Morazánica [Morazan Weekend]"
             first_wednesday = date(year, OCT, 1)
```

Comment:
Why isn't just
`first_wednesday = date(year, OCT, 1) + rd(weekday=WE(+1))`
here?

Reply:
Looks legit, thanks for the suggestion!